js/index.js

```diff
@@ -1,7 +1,81 @@
+console.log("I'm ready!");
+
 // Iteration 1: Names and Input
+let hacker1 = "Jackie";
+console.log(`"The driver's name is ${hacker1}"`);
 
+let hacker2 = "Shahinya";
+console.log(`"The navigator's name is ${hacker2}"`);
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters.`
+  );
+} else if (hacker2.length > hacker1.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters.`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters!`
+  );
+}
+
+// Iteration 3: Loops
+// 3.1 Print all characters of Driver's name, uppercase and with spaces in between.
+let spacedName = "";
+for (let i = 0; i < hacker1.length; i++) {
+  spacedName += hacker1[i].toUpperCase() + " ";
+}
+console.log(spacedName);
+
+// 3.2 Print all the characters of the navigator's name, in reverse order.
+let reversedName = "";
+for (let i = hacker2.length - 1; i >= 0; i--) {
+  reversedName += hacker2[i];
+}
+console.log(reversedName);
+
+// 3.3 Alphabetical order
+let hackerNames = [hacker1, hacker2];
+hackerNames.sort();
```

Comment:
I would be careful to use sort when wanting to sort by lexicographical order. This will not always work because it's not actually sorting depending on the alphabet. Best is to use localeCompare().